.idea/assetWizardSettings.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please remove from the branch all the .idea/* and .gradle/* files, these are local settings that shouldn't be pushed

Reply:
done